### DISCUSSION TITLE
Mobile play

### DISCUSSION THREAD

**Discussion by Killk06** [godotengine/godot-proposals#6538] [Platforms]
It would be so helpful if you guys made it so that you could test the game on your phone if you don't have a computer, and If you added stuff to make game development easier on your phone

**Comment by Zireael07** [comment]
1) You can already run the game on the phone (Godot automates the adb debug stuff)
2) There is an Android version of editor already

**Comment by AThousandShips** [comment]
I don't know that it's possible to export the game properly on Android because of the required build tools, I think I've seen it mentioned that the Android editor doesn't support it, I think this is a limitation not made by us but by the available software, could even be due to security features

**Reply by AThousandShips** [reply]
Here's some info: https://godotengine.org/article/android_godot_editor_play_store_beta_release/

And godotengine/godot#73130
